source/features/clean-conversation-headers.tsx

```diff
@@ -24,7 +24,8 @@ async function cleanPrHeader(byline: HTMLElement): Promise<void> {
 	// Extra author name is only shown on `isPRConversation`
 	// Hide if it's the same as the opener (always) or merger
 	const shouldHideAuthor = pageDetect.isPRConversation() && $('.author', byline)!.textContent === (await elementReady('.TimelineItem .author'))!.textContent;
```

Comment:
We should merge the logic into this. You can place `&& .closest()` (without null check) after `isPRConversation`